app/views/job_templates/_job_template_callback_tab_content.html.erb

```diff
@@ -0,0 +1,3 @@
+<div class="tab-pane" id="ansible_callback_enabled">
+  <%= checkbox_f f, :ansible_callback_enabled, :label => _('Enable Ansible Callback'), :disabled => @template.locked? %>
```

Comment:
@adamruzicka do we need to add translations for that? (I've never done this so I'm not sure what it includes).

Reply:
Add translations as in "mark the strings for translation"? Yes

Usually marked strings are extracted by the maintainer outside of the changes which introduced the new strings so if you meant extracting strings or even translating them then that would be no